spatial-operations/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorPrimary">#3F51B5</color>
+    <color name="colorPrimaryDark">#303F9F</color>
+    <color name="colorAccent">#FF4081</color>
+</resources>
```

Comment:
Done. Although the accent isn't really used as-is - could be useful if sample is extended and a FAB added or something like that?